src/backend/joanie/core/api/client.py

```diff
@@ -591,6 +593,13 @@ def get_queryset(self):
         )
         return queryset.filter(order__owner__username=username)
 
+    @extend_schema(
```

Comment:
You could to do the same for the method `download` of `ContractViewSet` and `preview` of `ContractDefinitionViewSet`

- https://github.com/openfun/joanie/blob/27024f98e83f9aa941a7b3ebf80f6bd644922268/src/backend/joanie/core/api/client.py#L1184-L1210
- https://github.com/openfun/joanie/blob/27024f98e83f9aa941a7b3ebf80f6bd644922268/src/backend/joanie/core/api/client.py#L1022-L1061